mino/minogrpc/controller/mod_test.go

```diff
@@ -418,6 +422,31 @@ func (badReader) Read([]byte) (int, error) {
 	return 0, fake.GetError()
 }
 
-type badCurve struct {
-	elliptic.Curve
+type badCurve struct{}
+
+func (b badCurve) Params() *elliptic.CurveParams {
+	return &elliptic.CurveParams{
+		Name: "BadCurve",
+		N:    big.NewInt(65536),
+	}
+}
+
+func (b badCurve) IsOnCurve(x, y *big.Int) bool {
+	return false
+}
+
+func (b badCurve) Add(x1, y1, x2, y2 *big.Int) (x, y *big.Int) {
+	panic("unsupported")
```

Comment:
would being more verbose help here ? Like "bad curve is intentionally unsupported".

Reply:
In this case it's more of a "not implemented" , because I didn't need to for my test , but point taken 😅